modules/behavioural_qc/jsx/tabs_content/incompleteForms.js

```diff
@@ -172,6 +172,18 @@ class IncompleteForms extends Component {
           ),
         },
       },
+      {
+        label: 'Data Entry Type',
+        show: true,
+        filter: {
+          name: 'Data Entry Type',
+          type: 'select',
+          options: {
+            'FDE': 'First Data Entry (FDE)',
```

Comment:
I think "SDE" has been the more common term in LORIS for the single data entry? Maybe add something to the LORIS meeting?